sklearn/lda.py

```diff
@@ -53,7 +53,7 @@ def _cov(X, shrinkage=None):
         if shrinkage == 'auto':
             sc = StandardScaler()  # standardize features
             X = sc.fit_transform(X)
-            s = sc.std_ * ledoit_wolf(X)[0] * sc.std_  # scale back
+            s = sc.std_ * ledoit_wolf(X)[0] * sc.std_[np.newaxis].T  # rescale
```

Comment:
Order does not matter in this case. It is only important that both, a column and a row vector are used.